benchmark/browser/scripts/benchmark.js

```diff
@@ -8,7 +8,7 @@ const http = require('http');
 
 const PORT = 1122;
 
-async function delay(ms) {
+function sleep(ms) {
```

Comment:
Only for naming consistency. Yes, Rx.js uses https://rxjs.dev/api/operators/delay.